impl/src/main/java/org/jboss/weld/bootstrap/events/ProcessSimpleInjectionTarget.java

```diff
@@ -45,6 +45,7 @@ public ProcessSimpleInjectionTarget(BeanManagerImpl beanManager, AnnotatedType<X
     public void setInjectionTarget(InjectionTarget<X> injectionTarget) {
         checkWithinObserverNotification();
         this.injectionTarget = injectionTarget;
+        BootstrapLogger.LOG.setInjectionTargetCalled(getReceiver().getClass(), injectionTarget.getClass());
```

Comment:
`toString()` may be overriden showing some important info. Otherwise `Object.toString()` always contains the class name.